src/vm.rs

```diff
@@ -73,6 +73,45 @@ impl MmioOpcode {
     }
 }
 
+#[cfg(test)]
+mod tests {
```

Comment:
similar to previous commit, the `mod tests` generally goes at the bottom of the file.

Reply:
fixed